### DISCUSSION TITLE
What's the quickest way to push a commit to main?

### DISCUSSION THREAD

**Discussion by rohitpaulk** [gitbutlerapp/gitbutler#5729] [Q&A]
Currently, I'm doing the following: 

- Make the change
- Drag to a new virtual branch
- Push
- Create pull request
- Merge pull request

Is there a simpler way to just push a change to main? We often have to make simple changes like a version bump / readme change and don't need to go through the whole PR process for those.

**Comment by wesharper** [comment]
What happens if you just name your virtual branch `main`?

**Reply by wesharper** [reply]
I could see some real drawbacks to using GitButler for pushing directly to main though. Since it likes to force push, it might be dicey.

**Reply by Byron** [reply]
By now it pushes with `--force-with-lease` which should prevent accidentally overwriting anything on the remote.

**Comment by Byron** [comment]
Currently there is no way to do that directly.

Personally I'd love if there was a 'stupid client' mode which forgoes some features that require `gitbutler/workspace` while allowing to interact with branches directly.
